app/src/main/kotlin/org/mifos/mobile/cn/ui/base/MifosBaseActivity.kt

```diff
@@ -47,7 +47,7 @@ open class MifosBaseActivity : AppCompatActivity(), BaseActivityCallback {
     override fun setContentView(layoutResID: Int) {
         super.setContentView(layoutResID)
        // toolbar = findViewById(R.id.toolbar)
-        setSupportActionBar(toolbar)
+       // setSupportActionBar(toolbar)
```

Comment:
Uncomment it, This line is enabling the support action bar in base activity.

Reply:
@therajanmaurya  since there is no toolbar yet, hence I got an error if I try to extend this activity because we were calling setSupportActionBar on an uninitialized toolbar.